quiche/src/recovery/congestion/delivery_rate.rs

```diff
@@ -210,6 +215,22 @@ struct RateSample {
     rtt: Duration,
 }
 
+impl RateSample {
+    const fn new() -> Self {
+        RateSample {
+            bandwidth: Bandwidth::zero(),
+            is_app_limited: false,
+            interval: Duration::ZERO,
+            delivered: 0,
+            prior_delivered: 0,
+            prior_time: None,
+            send_elapsed: Duration::ZERO,
+            ack_elapsed: Duration::ZERO,
+            rtt: Duration::ZERO,
+        }
+    }
+}
```

Comment:
`RateSample` was auto-deriving `Default` before, now it will be via `new()`

I specifically didn't want to auto-devire `Default` for `Bandwidth` since that could lead to subtle errors. Instead Bandwidth should be created via explicit methods such as `zero`, `infinity`...